wallet/src/de/schildbach/wallet/ui/dashpay/WalletBalanceBasedLiveData.kt

```diff
@@ -1,60 +0,0 @@
-package de.schildbach.wallet.ui.dashpay
-
-import android.text.format.DateUtils
-import androidx.core.content.ContextCompat
-import androidx.lifecycle.LiveData
-import de.schildbach.wallet.WalletApplication
-import de.schildbach.wallet.util.ThrottlingWalletChangeListener
-import org.bitcoinj.wallet.Wallet
-
-abstract class WalletBalanceBasedLiveData<T>(val walletApplication: WalletApplication = WalletApplication.getInstance())
-    : LiveData<T>() {
-
-    companion object {
-        private const val THROTTLE_MS = DateUtils.SECOND_IN_MILLIS
-    }
-
-    private var listening = false
-
-    private val wallet: Wallet
-        get() = walletApplication.wallet!!
```

Comment:
This causes a crash on the wallet reset. I removed the live data entirely to simplify this and fix the crash.